source/agora/test/Ledger.d

```diff
@@ -89,7 +90,10 @@ unittest
 /// test catch-up phase after initial booting (periodic catch-up)
 unittest
 {
-    TestConf conf = { topology : NetworkTopology.OneValidator };
+    TestConf conf = {
+        topology : NetworkTopology.MinimumValidators,
+        nodes : NetworkTopology.MinimumValidators + 1
```

Comment:
I wonder if it would make sense to split this into two:

```
extra_full_nodes
extra_validators
```

And then remove the `nodes` field. This way we can remove the need for `OneNonValidator` and also `MinimumValidators`, because `Simple` can be changed to do the same thing as `MinimumValidators` plus instantiate any extra nodes set in `extra_full_nodes` and `extra_validators`. 

This way we would always have at least `Enrollment.MinValidatorCount` validators in the genesis block.  Does it make sense to you?

Reply:
Oh, that's my mistake. I'll fix it.